src/main/java/com/piekoszek/nowaksharedrent/apartment/ApartmentServiceImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.piekoszek.nowaksharedrent.apartment;
+
+import com.piekoszek.nowaksharedrent.dto.User;
+import com.piekoszek.nowaksharedrent.dto.UserApartment;
+import com.piekoszek.nowaksharedrent.dto.UserRepository;
+
+import java.util.UUID;
+
+class ApartmentServiceImpl implements ApartmentService {
+
+    private ApartmentRepository apartmentRepository;
+    private UserRepository userRepository;
+
+    ApartmentServiceImpl(ApartmentRepository apartmentRepository, UserRepository userRepository) {
+        this.apartmentRepository = apartmentRepository;
+        this.userRepository = userRepository;
+    }
+
+    @Override
+    public void createApartment(String address, String city, String admin) {
+        Apartment apartmentToCreate = new Apartment(UUID.randomUUID().toString(), address, city, admin);
```

Comment:
Hmmm, I'd put ``UUID.randomUUID().toString()`` in some UUIDService, just like TimeService to make it more testable.